src/dotnet/project.json

```diff
@@ -28,7 +28,7 @@
         "Microsoft.DotNet.ProjectModel": "1.0.0-*",
         "Microsoft.DotNet.Compiler.Common": "1.0.0-*",
         "Microsoft.DotNet.Cli.Utils": "1.0.0-*",
-        "Microsoft.DotNet.ILCompiler.SDK": "1.0.4-prerelease-00006",
+        "Microsoft.DotNet.ILCompiler.SDK": "1.0.5-prerelease-00002",
```

Comment:
Thanks for the heads up.  AFAIK, the 1.0.5 version is already in our dependency feed, since I was working in master.

FYI - I'm pretty close to merging my change to remove www.myget.org from our NuGet.Config file.